src/any_llm/providers/minimax/minimax.py

```
@@ -45,7 +45,7 @@ def _convert_completion_response_async(
         async def chunk_iterator() -> AsyncIterator[ChatCompletionChunk]:
             async for chunk in response:
                 if isinstance(chunk, OpenAIChatCompletionChunk):
-                    if chunk.choices and chunk.choices[0].delta:
+                    if (chunk.choices and chunk.choices[0].delta) or chunk.usage is not None:
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)